src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -531,7 +533,11 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
 
     this.questionSubmissionForms.forEach((questionSubmissionFormModel: QuestionSubmissionFormModel) => {
       let isQuestionFullyAnswered: boolean = true;
-
+      if (!questionSubmissionFormModel.isValid) {
+        failToSaveQuestions[questionSubmissionFormModel.questionNumber] =
```

Comment:
I think this might not be possible, since the validity of each question in a submission is independent of each other, and maybe they might still want to save the questions for which are already valid